healthcheck/server.go

```diff
@@ -21,6 +21,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	time2 "github.com/polarismesh/polaris-server/common/time"
```

Comment:
time2 is recommended to use other names instead, such as commontime, etc.

Reply:
DONE